### PR TITLE
Install clang

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -77,17 +77,21 @@ stages:
...
     - script: |
-        yarn bootstrap
-        set CI=false
-        yarn build
-      name: Build
-    - script: |
-        set CI=true
-        yarn test
-      name: Test
+        powershell -Command "$ProgressPreference = 'SilentlyContinue'; iwr -outf %TEMP%\LLVM-8.0.0-win64.exe https://rust-lang-ci2.s3.amazonaws.com/rust-ci-mirror/LLVM-8.0.0-win64.exe"
```
Comment (katat) [inline azure-pipelines.yml:81]: I am wondering if it makes sense to add this new script above the original script of `install lerna`, and keep the rest of the scripts as it is without removing them adding the `pwsh` script.

Comment (Keith-CY) [reply azure-pipelines.yml:81]: I'll restore them if this script works.

**Comment by Keith-CY** [comment]
Seems good, I will pick this to the target branch.

**Comment by katat** [comment]
🎆  🍾

**Pull request closed without merging**